.github/workflows/generate-integrations-docs.yml

```
@@ -0,0 +1,47 @@
+---
+  # CI workflow used to generate documentation from integrations/integrations.js.
+
+name: Generate Go Integrations Documentation
+on: 
+  workflow_dispatch:
```

```suggestion
  workflow_dispatch: null
```

This is not strictly required by GHA, but it makes a number of YAML linting tools happier, and is consistent with what we do in workflows in the agent repo when a trigger needs no other configuration options.